docs/Azure_IoT_Hub_integration/Device_operations/Upgrading_firmware.md

```diff
@@ -0,0 +1,117 @@
+# Upgrading device firmware
+
+If you would like to upgrade the firmware of devices using the Azure IoT Hub, follow the instruction below.
+
+## Prerequisites
+
+ - At least one device with active Coiote DM - Azure IoT Hub [synchronization](../Importing_devices_to_Coiote_DM).
+
+## Invoking the *scheduleFirmwareUpdate* direct method
+
+To initiate the firmware upgrade procedure for your device:
+
+1. Go to your Azure hub account and under **Explorers**, select **IoT devices**.
+2. From the list, choose the device for which you want to upgrade the firmware.
+3. In the device view, select the **Direct Method** tab.
+
+    ![Invoking a FOTA direct method](images/direct_method_tab.png "Direct Method tab")
+
+4. Provide data for the following fields:
+
+     ![FOTA direct method configuration](images/direct_method_config.png "Direct Method config")
+
+    - **Method Name** - paste the `scheduleFirmwareUpdate` direct method name here.
+    - **Payload** - use the following payload with firmware upgrade parameters (remember to replace the example values where needed):
+          ```
+          {
+          	"name": "anjay-firmware",
+          	"firmwareUrl": "https://repository.avsystem.com/artifactory/gitlfs/demo.fw-pkg",
```

Comment:
I think we should not use repository.avsystem.com in examples. It is only our internal service.

Reply:
sure, changed to a more placeholder-like example